### DISCUSSION TITLE
How to install v5

### DISCUSSION THREAD

**Discussion by marius-bucur** [phalcon/phalcon#233] [General] [answered]
As the title says, I've tried to install v5 with no success.
Tried on Debian 11 and 10. 
On Deb11 Pecl ends up in an error, with libpcre3-dev installed.
On Deb10 both methods gives 4.1.

P.S. as per this page https://docs.phalcon.io/5.0/en/installation this is the default stable.

**Comment by niden** [comment, marked as answer]
Phalcon 5 is in beta2 for now, soon to become stable. You can install it using PECL

`pecl install phalcon-5.0.0beta2`

Once it becomes stable you will be able to pull it from relevant repositories.

**Reply by marius-bucur** [reply]
Thank you.

**Reply by Redweiweb** [reply]
downloading psr-1.2.0.tgz ...
Starting to download psr-1.2.0.tgz (28,238 bytes)
...done: 28,238 bytes
27 source files, building
running: phpize
sh: 1: phpize: not found
ERROR: `phpize' failed

**Reply by Jeckerson** [reply]
> phpize: not found

You need to install `php-dev` package.
